src/nvim/version.c

```diff
@@ -74,86 +74,92 @@ static char *features[] = {
 
 // clang-format off
 static int included_patches[] = {
+  // 922,
+  // 921 NA
+  // 920 NA
+  // 919 NA
+  // 918 NA
+  // 917,
   916,
   // 915,
   // 914,
-  // 913,
+  // 913 NA
   // 912,
-  // 911,
-  // 910,
+  // 911 NA
+  // 910 NA
   // 909,
-  // 908,
-  // 907,
-  // 906,
+  // 908 NA
+  // 907 NA
+  // 906 NA
   // 905,
   // 904,
   // 903,
-  // 902,
+  // 902 NA
   // 901,
-  // 900,
-  // 899,
+  // 900 NA
+  // 899 NA
   // 898,
   // 897,
   // 896,
   // 895,
-  // 894,
+  // 894 NA
   // 893,
   // 892,
   // 891,
-  // 890,
+  // 890 NA
   // 889,
   // 888,
   // 887,
-  // 886,
+  // 886 NA
   // 885,
-  // 884,
+  // 884 NA
   // 883,
   // 882,
   // 881,
   // 880,
-  // 879,
+  // 879 NA
```

Comment:
Why was this marked NA? It isn't specified in the commit message, and, looking at Nvim's commit log, it doesn't seem to be included.

Oh, and when you repush please don't amend the commit; I'll do that before merging.